### DISCUSSION TITLE
btrfs, ext4 etc - which to choose

### DISCUSSION THREAD

**Discussion by Rik44444** [SpiralLinux/SpiralLinux-project#339] [General]
Hi,
i just installed Spirallinux Plasma, which was a breeze. Great installer. I have 1 question, I loved the option of creating a swap for a hibernate option, it is something i have missed a lot. However, I was unsure which format to choose. I just went with the default, btrfs. Is this suitable for a beginner user like me? Or should I have opted for a different one?
OS is looking great so far, superfast! 
thanks,
Rik

**Comment by manujchandra** [comment]
As far as swap is concerned, default should be fine. It's set it and forget it.

As per my experience, just be aware of one thing, its near impossible to recover a BTRFS drive from errors. Once it's corrupted, you can run things like scrub etc. but chances of data recovery are slim.

This is what I do:
Laptop (daily driver is BTRFS)
Backup 1 is ext4 (encrypted)
Backup 2 is BTRFS (encrypted)
Backup 3 is on cloud (encrypted)

Keep backups on different technologies and locations.

**Reply by Rik44444** [reply]
ok thanks!
